### PR TITLE
initial r-perfmeas recipe

### PR DESCRIPTION
**Pull request by visze**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by visze** [comment]
package not directly related to biological science. But it includes bioconductor packages which are only available on bioconda. So conda-forge will not be possible.

However this package is needed for another R package of biological science `r-hemdag`

**Comment by daler** [comment]
Thanks @visze, agreed it should be here due to the bioconductor deps.

Can you please remove the Windows-related lines, as we don't build for Windows in bioconda?

**Comment by visze** [comment]
sure! I forgot to remove them because I tried it first on conda-forge :-)

File: recipes/r-perfmeas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% set version = '1.2.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
```
Comment (daler) [inline recipes/r-perfmeas/meta.yaml]: These too please?

**Pull request merged**